src/templates/drawerDeprecated.js

```diff
@@ -459,8 +478,8 @@ class DrawerDeprecated extends Component {
         const { onCloseComplete, position, onClickOutside } = this.props;
         const animationEvent = this._transitionProps(this.drawerContainerRef);
         const body = document.body;
-        const drawerLength = document.querySelectorAll('.ui.drawer').length;
-        const drawerDeprecatedLength = document.querySelectorAll('.ui.drawer.deprecated').length;
+        const drawerLength = document.querySelectorAll('.ui.drawer-is_modal').length;
```

Comment:
If we elect to rename the corresponding variable in the other file, let's please do it here as well.

Reply:
Renamed; resolving